policyengine_us/variables/gov/states/ar/tax/income/ar_taxable_capital_gains.py

```diff
@@ -25,14 +28,14 @@ def formula(person, period, parameters):
         capped_net_cap_gain = min_(net_capital_gain, p.exempt.cap)
         # Line 8 - Tax rate applied to capital gain
         taxable_capped_net_cap_gain = capped_net_cap_gain * (1 - p.exempt.rate)
```

Comment:
LMK if this addresses it. Also note that this changes how capital losses are dealt with--they should be negative, but with a floor of the negative cap (an earlier version may have done this).
```suggestion
        # 50% exempt if a gain, otherwise entire loss.
        exempt_fraction = where(capped_net_cap_gain > 0, p.exempt.rate, 0)
        taxable_amount = capped_net_cap_gain * (1 - exempt_fraction)
        # Lines 9-11: Arkansas short term capital gain if any.
        stcg_if_any = max_(st_capital_gains, 0)
        # Line 12: Total taxable gain or loss. Loss is capped.
        total_taxable_cap_gain_or_loss = taxable_amount + stcg_if_any
        loss_cap = p.loss_cap[filing_status]
        return max_(-loss_cap, total_taxable_cap_gain_or_loss)
```

Reply:
Do we want the taxable capital losses to return as negative values? - this will cause issues as we add them to the total taxable income if I am not seeing this wrong

Reply:
yes, capital losses are negative capital gains...

does the form say to add the absolute value to taxable income?

Reply:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/0c883640-3c24-47c2-9f94-8f4bc85ac304)
[Form](https://www.taxformfinder.org/forms/2023/2023-arkansas-form-ar1000d.pdf#page=1)

Reading the above, it seems like capital loss reduces taxable income (up to $1,500) - so I agree with the code suggestion, unless I am misreading it.

Reply:
Filling out the tax form, this is what I am getting when considered a person with no long term capital gains and only short term losses:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/9b88f133-5cc6-4130-addc-48343776062c)

Did I miss anything when filling this out? I would make slight adjustments to the code if not

Reply:
thanks - believe the form aligns with my code suggestion

Reply:
Not quite, in the code suggestion we use:
`        exempt_fraction = where(capped_net_cap_gain > 0, p.exempt.rate, 0)
`
Which only provides the 50% reduction rate if we have net capital gains - I will change it so it also applies to the capital loss situation

Reply:
Sorry I misread. Why multiply 7b by 50%? It's not a gain.

Reply:
I see, adjusted, my mistake